src/test/java/core/basesyntax/operations/BalanceHandlerTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.operations;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class BalanceHandlerTest {
+    private static final FruitTransaction CORRECT_FRUIT_TRANSACTION =
+            new FruitTransaction(FruitTransaction.Operation.BALANCE, "banana", 100);
+    private static final FruitTransaction NULL_FRUIT_TRANSACTION = null;
+    private static final FruitTransaction INCORRECT_FRUIT_TRANSACTION =
+            new FruitTransaction(FruitTransaction.Operation.PURCHASE, "apple", 50);
```

Comment:
instantiate objects to test inside test method

any reasons to make all things constants

The same for further tests.
```suggestion
```